### PR TITLE
Issue 251 Add photo change visibility to hidden

### PR DESCRIPTION
**Pull request by angry010101**
Implemented
* Openplacereviews files from pull request 10178 copied
https://github.com/osmandapp/OsmAnd/pull/10178/files#diff-95b72b4f22c3469cb18389309559479e3e1ab3fa147df7c232079197bbf7725dR319

* add photo hidden

### CODE REVIEW

File: OsmAnd/src/net/osmand/plus/mapcontextmenu/MenuBuilder.java [hunk 1]
```diff
@@ -336,6 +336,8 @@ public void onClick(View view) {
 		b.setTypeface(null, Typeface.BOLD);
 		b.setText(context.getResources().getString(R.string.shared_string_add_photo));
 		b.setBackgroundResource(R.drawable.btn_border_light);
+		//TODO feature under development
+		b.setVisibility(View.GONE);
```
Comment (vshcherb) [inline OsmAnd/src/net/osmand/plus/mapcontextmenu/MenuBuilder.java:340]: Here should not be gone, it should be in separate request! And this request we should test with button present

File: OsmAnd/src/net/osmand/plus/mapcontextmenu/builders/cards/ImageCard.java [hunk 2]
```diff
@@ -509,6 +509,7 @@ private void getPicturesForPlace(List<ImageCard> result, long l) {
 					"Requesting location images...", false, false);
 			try {
 				if (!Algorithms.isEmpty(response)) {
+					//TODO extract place id
 					JSONArray obj = new JSONObject(response).getJSONArray("objects");
 					JSONArray images = ((JSONObject) ((JSONObject) obj.get(0)).get("images")).getJSONArray("outdoor");
```
Comment (vshcherb) [inline OsmAnd/src/net/osmand/plus/mapcontextmenu/builders/cards/ImageCard.java:514]: Here we need to fix pottential BUG! there is no clear evidence that 
```((JSONObject) obj.get(0))```
has no crash!

File: OsmAnd/src/net/osmand/plus/osmedit/opr/OPRImage.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package net.osmand.plus.osmedit.opr;
+
```
Comment (vshcherb) [inline OsmAnd/src/net/osmand/plus/osmedit/opr/OPRImage.java:2]: What do we need such small class? Could be emdedded to the service it's used

File: OsmAnd/src/net/osmand/plus/osmedit/opr/OpenDBAPI.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+	public int uploadImage(String[] placeId, String privateKey, String username, String image) throws FailedVerificationException {
+		if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+			Security.removeProvider("BC");
+			Security.addProvider(new BouncyCastleProvider());
+		}
+		KeyPair kp = SecUtils.getKeyPair(ALGO_EC, privateKey, null);
+		String signed = username;// + ":opr-web";
+
+		JsonFormatter formatter = new JsonFormatter();
+		OPRImage OPRImage = new GsonBuilder().create().fromJson(image, OPRImage.class);
```
Comment (vshcherb) [inline OsmAnd/src/net/osmand/plus/osmedit/opr/OpenDBAPI.java:42]: local variable ```OPRImage``` - guidelines?

File: OsmAnd/src/net/osmand/plus/mapcontextmenu/builders/cards/ImageCard.java (hunk 2 above)
Comment (angry010101) [reply OsmAnd/src/net/osmand/plus/mapcontextmenu/builders/cards/ImageCard.java]: try catchblock will handle the exception

**Pull request merged**